README.md

```diff
@@ -262,24 +220,225 @@ We can also pull data from different forms and join the various form data. Here
         },
 ```
 
+The `site-config.json` property `APP.supersetBase` should also be provided as described in the creating the config file section below
+
 <br>
 
 **Pulling Directly from Mapbox**
 
 Ensure that the data file for the layer is located in the specified source path `data/ken_health_sites.geojson or a remote url that points to the file`.
 
-### 3. Actions
+### 3. Create config file
+
+The application requires a `public/config/site-config.json` to initialize the Map and load `public/config/layers/*.json` configurations. Create a `public/config/site-config.json` which will hold the configurations. An example of a `site-config` configuration is given below
+
+```json
+{
+  "APP": {
+    "mapConfig": {
+      "container": "map",
+      "style": "mapbox://styles/mapbox/light-v9",
+      "center": [
+        36.25
+        0.34
+      ],
+      "zoom": 6
+    },
+    "accessToken": "<mapbox-studio-access-token>",
+    "appIcon": "/img/gisida-logo.png",
+    "appName": "GISIDA EXAMPLE",
+    "appColor": "#000",
+  },
+  "STYLES": [
+    {
+      "label": "Light",
+      "style": "mapbox://styles/mapbox/light-v9"
+    },
+    {
+      "label": "Streets",
+      "style": "mapbox://styles/mapbox/streets-v9"
+    },
+    {
+      "label": "Satellite",
+      "style": "mapbox://styles/mapbox/satellite-v9"
+    }
+  ],
+  "LAYERS": [
+    "ken-health-sites",
+  ]
+}
+```
+
+Let's go over each of the keys in the `site-config.json` file
+
+1. `APP` : The `APP` key expects and object and ontains properties for the site settings. The following are the properties supported
+
+- `mapConfig`: An object that contains keys for setting the values for the properties for
+  interacting with the map. More information about each of these properties including their defaults
+  can be found at the [Mapbox GL JS API Reference](https://docs.mapbox.com/mapbox-gl-js/api/map/)
+
+  - `container`: Your HTML element in which Mapbox GL JS will render the map
+  - `style`: The default Mapbox style to be loaded from the Mapbox API of the
+    form `mapbox://styles/:owner/:style`
+  - `center`: The initial geographical centerpoint of the map (Uses longitude, latitude coordinated order).
+  - `zoom` : The initial zoom level of the map.
+  -  `mapBounds` : Map points when exporting i.e from south west to north east.
+      e.g 
+
+      ```
+      {
+        "_sw": {
+          "lng": 22.13636132811544,
+          "lat": -0.14320447808067627
+        },
+        "_ne": {
+          "lng": 62.89563867185916,
+          "lat": 11.188804478371793
+        }
+      }
+      ```
+  - `preserveDrawingBuffer` : whether to preserve the buffers until manually cleared or overwritten.
+
+- `accessToken` : Your Mapbox API access token.
+- `hyperLink` : Add links and description to menu items. 
```

Comment:
> `hyperLink` : Add links and description to menu items. 

Let's mention each of the keys for this object and what they are used for such as `mapConfig` object above